system-variables.md

```diff
@@ -218,6 +236,16 @@ mysql> SELECT * FROM t1;
 - Default value: ""
 - The local unix socket file that the `tidb-server` is listening on when speaking the MySQL protocol.
 
+### sql_log_bin
+
+- Scope: SESSION | GLOBAL
+- Default value: `ON`
+- This indicates if changes should be written to the [TiDB Binlog](/tidb-binlog/tidb-binlog-overview.md) or not.
+
+> **Note:**
+>
+> Setting `sql_log_bin` as a global variable is not recommended. Future versions may only allow setting this as a session variable.
```

Comment:
```suggestion
> It is not recommended to set `sql_log_bin` as a global variable because the future versions of TiDB might only allow setting this as a session variable.
```